setup.cfg

```diff
@@ -64,7 +64,7 @@ install_requires =
     recommonmark>=0.7
     scikit_learn
     scipy
-    setuptools
+    setuptools<=57.5.0
```

Comment:
oh weird, typically setuptools isn't actually a requirement for using the code. I'm surprised this is listed here